sql/core/src/test/scala/org/apache/spark/sql/execution/datasources/json/JsonSuite.scala

```diff
@@ -871,7 +869,6 @@ class JsonSuite extends QueryTest with SharedSQLContext with TestJsonData {
               |WHERE _unparsed IS NOT NULL
             """.stripMargin),
           Row("{") ::
-            Row("") ::
```

Comment:
Yes, as the jira title shows I believe empty line should be skipped for json file. "{}" represent a json object with nothing. Empty line in json world is meaningless